app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
> I would prefer to keep the schema in a separate project.

👍 

> But then the question raises: how should we deliver the schema to the `diaspora_federation` gem?

I think a gem is the best solution, since I don't want that every pod needs to download the schema every time. We can also copy the schema (like we did for the node-info schema in diaspora), but I think a gem that we can add as dependency is better than copy the schema every time there is a new version.